app/services/renderer/products.rb

```diff
@@ -39,7 +40,8 @@ def self.props(item, product)
     end
   end
 
-  def self.basic_product(url_options, current_store, current_currency, item, product)
+  def self.basic_product(url_options, current_store_id, current_currency, item, product)
```

Comment:
why you replace current_store with current_store_id

Reply:
beacuse when i pass whole object by default it convert with to_s and look like #